sql-odbc/docs/dev/run_tests.md

```diff
@@ -0,0 +1,104 @@
+# ODFE SQL ODBC Driver Testing
+
+## Preparation
+
+* Latest version of [Open Distro for Elasticsearch](https://opendistro.github.io/for-elasticsearch-docs/docs/install/)
+
+### Loading Test Datasets
+
+Loading a dataset requires an [elasticsearch](https://opendistro.github.io/for-elasticsearch-docs/docs/install/) service running with [kibana](https://opendistro.github.io/for-elasticsearch-docs/docs/kibana/). If either of these are missing, please refer to the documentation on how to set them up.
+
+Note, if you wish to work with SSL/TLS, you need to configure Elasticsearch and Kibana to support it. See Working With SSL/TLS below.
+
+First load the sample datasets provided by kibana.
+
+1. Select home (top left corner)
+2. Select 'Load a data set and a Kibana dashboard'
+3. Select 'Add data' under 'Sample flight data'
+4. Select 'Add data' under 'Sample eCommerce orders'
+5. Select 'Add data' under 'Sample web logs'
+
+Then load custom data sets using the kibana console.
```

Comment:
Add a reference to required custom dataset `kibana_sample_data_types`

Reply:
added reference